diagnostic.md

```diff
@@ -73,14 +88,23 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl -H "Content-Type: application/json" -X POST -d '{"name": "French Cruller","price": "$0.99"}}' http://www.example.com/donuts
```

Comment:
Should look like this: 
`curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'`